src/server/acl/acl_family.cc

```diff
@@ -157,12 +163,33 @@ std::variant<User::UpdateRequest, ErrorReply> ParseAclSetUser(CmdArgList args) {
 
 }  // namespace
 
+void AclFamily::StreamUpdatesToAllProactorConnections(std::string_view user, uint32_t update_cat) {
+  auto update_cb = [user, update_cat]([[maybe_unused]] size_t id, util::Connection* conn) {
+    DCHECK(conn);
+    auto connection = static_cast<facade::Connection*>(conn);
+    auto ctx = static_cast<ConnectionContext*>(connection->cntx());
+    if (ctx && user == ctx->authed_username) {
+      ctx->acl_categories = update_cat;
+    }
+  };
+
+  pp_.AwaitFiberOnAll([this, update_cb](util::ProactorBase* pb) {
+    if (main_listener_) {
+      main_listener_->TraverseConnections(update_cb);
```

Comment:
```
  // traverses all client connections in all threads. cb must be thread safe.
  // cb should not keep Connection* pointers beyond the run of this function because
  // Connection* are valid only during the call to cb.
  void TraverseConnections(TraverseCB cb);
```

Reply:
Why is this an issue? `update_cb` is thread safe, since each connection can't be part of multiple `proactors` ? And since this is the case, then it's associated context can't also be part of multiple proactors ? That is we don't write on the same context simultaneously (there are no data races).

Am I missing anything?